### PR TITLE
Feature/removed deprecated warnings

### PR DESCRIPTION
**Pull request by UmangKesari**
Removed Deprecated warning for play 2.6

### CODE REVIEW

File: app/controllers/FeedbackFormsResponseController.scala [hunk 1]
```diff
@@ -235,6 +235,7 @@ class FeedbackFormsResponseController @Inject()(messagesApi: MessagesApi,
 
       (question.questionType, question.mandatory) match {
         case ("MCQ", true)      => if (question.options.contains(response) && response.nonEmpty) {
+          Logger.info(s"The response was -------------------------> " + response)
```
Comment (mavericksid) [inline app/controllers/FeedbackFormsResponseController.scala]: Can we get rid of this logger? It does not seem useful.

File: app/controllers/UsersController.scala [hunk 2]
```diff
@@ -168,13 +168,18 @@ class UsersController @Inject()(messagesApi: MessagesApi,
     )
   }
 
-  def login: Action[AnyContent] = action { implicit request =>
-    Ok(views.html.users.login(loginForm))
+  def login: Action[AnyContent] = action.async { implicit request =>
+    val username = configuration.get[String]("session.username")
```
Comment (mavericksid) [inline app/controllers/UsersController.scala]: Can we make "username" as a class level variable instead? It is not going to change often.

File: app/controllers/FeedbackFormsResponseController.scala (hunk 1 above)
Comment (UmangKesari) [reply app/controllers/FeedbackFormsResponseController.scala]: Yes.

**Comment by mavericksid** [comment]
This pull request is no longer required.

**Pull request closed without merging**